types/openfin/v49/openfin-tests.ts

```diff
@@ -0,0 +1,1001 @@
+function test_application() {
+    let application: fin.OpenFinApplication;
+    // constructor
+    application = new fin.desktop.Application({
+        url: "application.html",
+        uuid: "74BED629-2D8E-4141-8582-73E364BDFA74",
+        name: "Application Name",
+        plugins: false,
+        mainWindowOptions: {
+            defaultHeight: 600,
+            defaultWidth: 800,
+            defaultTop: 300,
+            defaultLeft: 300,
+            autoShow: true
+        }
+    }, (successObj) => {
+        console.log("Application successfully created, HTTP response code:", successObj);
+        application.run();
+    }, (error) => {
+        console.log("Error creating application:", error);
+    });
+	// createFromManifest
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/openfin/v49/openfin-tests.ts;
  Line: 22;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.